src/Altinn.Apps/AppTemplates/AspNet/App/Program.cs

```diff
@@ -12,9 +14,27 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder =>
-                {
-                    webBuilder.UseStartup<Startup>();
-                });
+             .ConfigureWebHostDefaults(webBuilder =>
```

Comment:
Is there a simpler notation to make the file less complex for the app developers?